storage/sql/sql.go

```diff
@@ -3,11 +3,10 @@ package sql
 
 import (
 	"database/sql"
+	"github.com/dexidp/dex/pkg/log"
```

Comment:
nit: this should be in a separate import block from the stdlib

import (
    "database/sql"

    "github.com/dexidp/dex/pkg/log"

    // third party drivers
    // ...
)

Reply:
Fixed